src/components/internal/Popover.tsx

```diff
@@ -11,7 +11,7 @@ interface PopoverProps extends React.PropsWithChildren<any> {
 
 /** Popover is used for generating menus and list-boxes */
 export function Popover(props: PopoverProps) {
```

Comment:
**question:** Did we find out if we could get rid of React-Popper and use this? Does this handle changing positions when hitting the edges of the screen?

Reply:
Yes, this does handle changing positions. We could get rid of React-Popper, but we might want to generalize this component a bit more before then. Part of some tech debt.

Reply:
Nice! Great find!